src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    line-height: 14px;
+    padding: $product-img-padding 0;
+
+    &-reviews {
+      margin: 0;
+      font-size: $font-S;
+      color: $title-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 4px;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-last-child(1n + 2) {
```

Comment:
Should be the same functionality as in the stars task
<img width="824" alt="image" src="https://github.com/user-attachments/assets/e14a0da0-b508-43dd-9cc4-720157319fa6">